tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +147,18 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert len(jesse.inventory) == len(tai.inventory)
```

Comment:
See comment above. Is this test concerned with knowing that the length of these two lists are the same? Is that part of the logic of the method that is under test here?

Same comment applies to similar assertions written in the tests below.

Reply:
✅ Fixed (will push later!)